tests/functions/test_drop_constant_columns.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+
+@pytest.mark.functions
+def test_row_to_names(df_constant_columns):
```

Comment:
I think I know where this function name came from, @fireddd :smile:, and I think the test function's name isn't right :wink:.

```suggestion
def test_drop_constant_columns(df_constant_columns):
```

Reply:
Not my proudest moment :(